vcbuild.bat

```diff
@@ -117,7 +117,7 @@ if "%i18n_arg%"=="full-icu" set configure_flags=%configure_flags% --with-intl=fu
 if "%i18n_arg%"=="small-icu" set configure_flags=%configure_flags% --with-intl=small-icu
 if "%i18n_arg%"=="intl-none" set configure_flags=%configure_flags% --with-intl=none
 if "%i18n_arg%"=="without-intl" set configure_flags=%configure_flags% --without-intl
-if "%engine%"=="chakracore" set configure_flags=%configure_flags% --without-intl --without-inspector
+if "%engine%"=="chakracore" set configure_flags=%configure_flags% --without-intl --without-inspector --without-v8-platform --without-bundled-v8
```

Comment:
> The gyp changes actually makes my life easier (on Linux) 

I see. In that case, I will try to condense all the v8 conditions under 1 single condition. In case, I don't get time today, i will send separate PR for this.